docker/labkey-20.3-community/README.md

```diff
@@ -0,0 +1,48 @@
+samples/docker/labkey-20.3-community
+==========
+
+This folder contains a sample docker-compose file and sample Docker configurations to stand up an instance of LabKey Server Community edition for testing.
+
+### IMPORTANT: Not Recommended for Production use
+ This image was meant for trying out LabKey Server and not meant for running a production server or for storing important biomedical data. 
+
+#### Technology configuration:
+* LabKey Server 20.3.2 
+* Tomcat 9
+* OpenJDK-14
+* PostgreSQL 11
+
+
+## Usage 
+
+### Recommended configuration steps
+
+1. Clone this repo
+1. Edit the labkey/config/labkey.xml and choose your own settings for the following
+    1. jdbc (DB) username and password
+    1. MasterEncryptionKey password 
+    1. SMTP settings
+1. Edit the postgres/fixtures/create_fixtures.sql file
+    1.  DB username and password should match those configued in labkey/config/labkey.xml 
+
+
+### Running LabKey Server 
+
+To build and run the LabKey service  
+
+    sudo docker-compose up -d --build
+
+After few seconds, open [http://localhost:8080/labkey](http://<host>:8080) to see the LabKey Server initialization page.
```

Comment:
I think you want `[http://localhost:8080/labkey](http://localhost:8080/labkey)` which will be a link to "http://localhost:8080/labkey".

Reply:
Fixed